learning_journal/templates/journal.jinja2

```diff
@@ -0,0 +1,11 @@
+{% extends "layout.jinja2" %}
+
+
+{% block content %}
+    <article>
+      <h2>{{ entry.title }}</h2>
+      <h6>{{ entry.date }}</h6>
+      </p>{{ entry.body }}</p>

```

Comment:
I think I noted this on your step 2 submission but this is the ending p-tag. You want a beginning p-tag here.